batch.py

```diff
@@ -162,9 +162,10 @@ def process_feed(args, callback=None):
     # get mutation info
     locator = SC2Locator()
     mutations = {}
+    print(get_mutations(by_lineage).items())
```

Comment:
print statements should be removed before code can be merged to production branch